components/remote/ui/ir/CardButtonsTransport.qml

```diff
@@ -29,91 +31,118 @@ Item {
     width: parent.width - 60
     anchors.horizontalCenter: parent.horizontalCenter
 
-    Flow {
+    ColumnLayout {
+        anchors {
+            top: parent.top
+            horizontalCenter: parent.horizontalCenter
+        }
         width: parent.width - 60
-        anchors { top: parent.top; horizontalCenter: parent.horizontalCenter }
         spacing: 32
 
-        Button {
-            visible: obj.isSupported(Remote.F_PLAY)
-            title: qsTr("Play") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.play(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_PAUSE)
-            title: qsTr("Pause") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.pause(); }
-        }
-
-        Button {
-            visible: obj.isSupported(Remote.F_STOP)
-            title: qsTr("Stop") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.stop(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            IconButton {
+                visible: obj.isSupported(Remote.F_RECORD)
+                image.source: "qrc:/images/remote/fiber_manual_record-white-48dp.svg"
```

Comment:
For icons, best would be to extend the `icons.tft` font file with new icons and reference them from `Style.qml`.